Components/Hero.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+
+import '../css/Components/Hero'
+
+import Title from './Title';
+import SubSectionTitle from './SubSectionTitle';
+
+import PlayIcon from '../images/play.svg';
+import TwitterLogo from '../images/twitter.svg';
+import YoutubeLogo from '../images/youtube.svg';
+import FacebookLogo from '../images/facebook.svg';
+
+export default class Hero extends Component {
+  render() {
+    return <section className="Hero">
+      <div className="Hero-background">
+      </div>
+
+      <div className="Hero-foreground">
+        <div className="Hero-content">
+          <div className="Hero-title">
+            <SubSectionTitle>Mirror Conference 2.0</SubSectionTitle>
```

Comment:
The assumptions are correct, the naming was a suggestion @fcbaila gave and in fact, these will only be used on sections. The only exception is here on the `Hero` and I'm going to create these classes on the `Hero` component, as we already discussed in person.